plugins/services/src/js/components/modals/NewCreateServiceModal.js

```diff
@@ -158,6 +177,11 @@ class NewServiceFormModal extends Component {
     this.setState(this.getResetState());
   }
 
+  handleClearErrors() {
+    this.props.clearError();
+    this.setState({showAllErrors: false});
```

Comment:
Let's not set this unless we are navigating back to the form from picking another option or editing a service